src/status_im/contexts/wallet/send/transaction_confirmation/view.cljs

```diff
@@ -246,19 +238,29 @@
                                        :margin-top          (safe-area/get-top)
                                        :background          :blur
                                        :accessibility-label :top-bar}]
-           :footer                   (when (and route (seq route))
-                                       [standard-auth/slide-button
-                                        {:size                :size-48
-                                         :track-text          (if (= transaction-type :tx/bridge)
-                                                                (i18n/label :t/slide-to-bridge)
-                                                                (i18n/label :t/slide-to-send))
-                                         :container-style     {:z-index 2}
-                                         :customization-color account-color
-                                         :on-auth-success     #(rf/dispatch
-                                                                [:wallet/send-transaction
-                                                                 (security/safe-unmask-data
-                                                                  %)])
-                                         :auth-button-label   (i18n/label :t/confirm)}])
+           :footer                   [:<>
+                                      [transaction-details
+                                       {:estimated-time-min estimated-time-min
+                                        :max-fees           fee-formatted
+                                        :token-display-name token-display-name
+                                        :amount             amount
+                                        :to-network         bridge-to-network
+                                        :theme              theme
+                                        :route              route
+                                        :transaction-type   transaction-type}]
```

Comment:
Designs changes, now we should display transaction details right above the slider